indico/htdocs/js/indico/modules/categories/management.js

```diff
@@ -59,6 +56,18 @@
             });
         });
 
+        $bulkMoveButton.on('click', function(evt) {
+            evt.preventDefault();
+            if ($(this).hasClass('disabled')) {
```

Comment:
We use `$this` below because we need a reference to the element within the `success` callback.